core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/CASLockFactory.java

```diff
@@ -127,12 +130,22 @@ private CASLockFactory(final Builder builder)
 
         mySession = builder.myNativeConnectionProvider.getSession();
         myRemoteRouting = builder.myNativeConnectionProvider.getRemoteRouting();
+        mySerialConsistency = builder.myNativeConnectionProvider.getSerialConsistency();
 
         verifySchemasExists();
 
-        ConsistencyLevel serialConsistencyLevel = myRemoteRouting
+        if ("DEFAULT".equals(mySerialConsistency))
+        {
+            serialConsistencyLevel = myRemoteRouting
+                ? ConsistencyLevel.LOCAL_SERIAL
+                : ConsistencyLevel.SERIAL;
+        }
+        else
+        {
+            serialConsistencyLevel = "LOCAL".equals(mySerialConsistency)
```

Comment:
Why not use a regular constant for this?

Reply:
Done